time/time.go

```diff
@@ -27,7 +28,7 @@ func (t *Time) MarshalJSON() ([]byte, error) {
 // The time is expected to be in Deis' datetime format.
 func (t *Time) UnmarshalText(data []byte) (err error) {
```

Comment:
can you remove the named return param here? it's shadowed below and hence not used